### ISSUE TITLE
Regression on isUtf8MimeType

### ISSUE THREAD

**Issue by samson-sham** [fastify/fastify-static#382]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the bug has not already been reported

### Fastify version

4.17.0

### Node.js version

18.7.0

### Operating system

macOS

### Operating system version (i.e. 20.04, 11.3, 10)

13.3.1

### Description

Occurred on both MacOS and Linux (alpine), app fatally crashed when requesting static content using @fastify/static.
No errors on @fastify/send@2.0.1

### Steps to Reproduce

```
/app/node_modules/@fastify/send/lib/isUtf8MimeType.js:4
  const len = value.length
                    ^

TypeError: Cannot read properties of null (reading 'length')
    at Function.isUtf8MimeType (/app/node_modules/@fastify/send/lib/isUtf8MimeType.js:4:21)
    at getContentType (/app/node_modules/@fastify/static/index.js:481:13)
    at PassThrough.<anonymous> (/app/node_modules/@fastify/static/index.js:152:40)
    at PassThrough.emit (node:events:513:28)
    at Readable.pipe (node:internal/streams/readable:812:8)
    at SendStream.stream (/app/node_modules/@fastify/send/lib/SendStream.js:797:10)
    at SendStream.send (/app/node_modules/@fastify/send/lib/SendStream.js:708:8)
    at onstat (/app/node_modules/@fastify/send/lib/SendStream.js:730:10)
    at FSReqCallback.oncomplete (node:fs:209:5)
```

### Expected Behavior

Value should be checked against falsy values and error should be handled.

**Comment by mcollina** [comment]
@Uzlopak could you take a look?

**Comment by Uzlopak** [comment]
I think the issue is fixed with #384
